src/rust/demikernel/libos/mod.rs

```diff
@@ -142,28 +139,22 @@ impl LibOS {
             #[cfg(feature = "catpowder-libos")]
             LibOSName::Catpowder => {
                 // TODO: Remove some of these clones once we are done merging the libOSes.
```

Comment:
Does this refer to the runtime.clone() below? Can that be avoided at all?

Reply:
I think I have removed all possible clones. I'm not sure that it is possible to remove the final set of clones as both the inetstack (as the network transport) and the libOS can create coroutines, so need access to the scheduler.